Commands/Reformat Document.tmCommand

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>beforeRunningCommand</key>
+	<string>saveActiveFile</string>
+	<key>command</key>
+	<string>#!/bin/bash
+
+[[ -f "$HOME/.rvm/scripts/rvm" ]] &amp;&amp; . "$HOME/.rvm/scripts/rvm"
+
+(
+bundler_dir="${TM_PROJECT_DIRECTORY:-$(dirname "$TM_FILEPATH")}"
+if [[ -f "$bundler_dir/Gemfile" ]]; then
+  cd "$bundler_dir" &amp;&amp; bundle list rubocop
+  if [[ $? == 0 ]]; then bundler_rubocop='bundle exec rubocop'; fi
+fi
+
+if [[ ! `which rubocop` &amp;&amp; -z "$bundler_rubocop" ]]; then
+  "$DIALOG" tooltip --text "This command requires RuboCop.
+Please install RuboCop via “gem install rubocop”."
+  exit
+fi
+
+aha_found=`which aha`
+if [[ $aha_found ]]; then
+  color='n';
+  format='--html'
+else
+  format='--text'
+fi
+
+rubocop="${TM_RUBOCOP:-"${bundler_rubocop:-rubocop}"}"
```

Comment:
I think the order should be:

1. The version of rubocop specified via Bundler
1. `TM_RUBOCOP`
1. The command `rubocop`

This way the most specific version of Rubocop for a project will be used. If you would run Rubocop form the terminal for a project using Bundler you would most likely run it with `bundle exec`. If you use `rubygems-bundler` it will have the same behavior just running `rubocop`. It would be a bit strange if running from inside of TextMate would use a different version.